packages/serverpod/lib/src/protocol/log_entry.yaml

```diff
@@ -3,7 +3,7 @@ class: LogEntry
 table: serverpod_log
 fields:
   ### The id of the session this log entry is associated with.
-  sessionLogId: int, relation(parent=serverpod_session_log)
+  sessionLogId: int, relation(parent=serverpod_session_log, onDelete=Cascade)
```

Comment:
Note for future reference - Before the introduction of the `relation` keyword these used the old `parent` notation and therefore had `Cascade` as onDelete action.

This addition therefore ensures that the old behavior is preserved even if we use the new `relation` keyword syntax.